frontend/src/components/common/header/popus.styles.scss

```diff
@@ -0,0 +1,10 @@
+.popup-arrow {
+  color: var(--color-header-background);
+  filter: brightness(1.2);
+}
+
+.popup-content {
+  position: absolute;
+  top: 20px;
+  left: 20px;
+}
```

Comment:
```suggestion
```
move it to `frontend/src/components/common/header/styles.module.scss`
just use `global` selector